src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -151,3 +156,66 @@ def signature_hash_algorithm(self):
             raise UnsupportedAlgorithm(
                 "Signature algorithm OID:{0} not recognized".format(oid)
             )
+
+    @property
+    def extensions(self):
+        # TODO: remove this when we finish landing support for extensions.
+        warnings.warn(
+            "Extension support is not fully implemented. Unsupported "
+            "extensions are IGNORED at this time."
+        )
```

Comment:
The CA false test (which uses the cryptography.io cert itself) and the test without basic constraints (which uses PKITS data from NIST) both contain critical key usage extensions that would cause failures. Once we land support for that extension we'd be okay though.

Alternately we could replace those vectors with more specific ones, but I'll need to submit a new vector PR for that.